db/seeds.rb

```diff
@@ -60,7 +60,7 @@
                         gpu: rand(50)+1,
                         ram: 4+rand(17),
                         size: 1+ rand(4),
-                        aasm_state: ["Released to arcade", "Not released", "Game under review", "Game not uploaded, it is not compatible with the system", "Rejected"].sample })
+                        aasm_state: ["under_review", "rejected", "unreleased", "released", "incompatible"].sample })
```

Comment:
Can you create a variables that holds these state that way you don't have to repeat yourself below? :neckbeard: